docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -0,0 +1,380 @@
+        contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvIGVuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlPTEK
+          verification: {}
+        mode: 0644
+```
+
+If you need to configure your deployment for real-time running or low latency, see the [performance addon operator documentation](performance-addon-operator), or you plan to use a technology such as DPDK that requires the use of [hugepages](huge-pages-perf-addon).
+
+Once your OpenShift control plane is running, you will need to install the SR-IOV Network Operator. To do so, you will need access to an account on your OpenShift cluster with `cluster-admin` privileges. Once you are logged into that account, follow [these steps](sriov-operator) to install the Operator. Then, [configure your SR-IOV network device](configure-sriov-network-device).
+
+## Creating SR-IOV Networks for Worker Nodes
+
+The SR-IOV networks that are attached to worker nodes need to be created before creating the nodes. Please reference the example of how to create radio and uplink provider networks in OpenStack.
+
+```sh
+# Create Networks
+openstack network create radio --provider-physical-network radio --provider-network-type vlan --provider-segment 120
+openstack network create uplink --provider-physical-network uplink --provider-network-type vlan --external
+
+# Create Subnets
+openstack subnet create --network radio --subnet-range <radio_network_subnet_range> radio
+openstack subnet create --network uplink --subnet-range <uplink_network_subnet_range> uplink
+```
+
+## Creating SR-IOV Worker Nodes in IPI
+
+Creating worker nodes as a post-install operation in IPI is handled by the machine-api. To create a new set of worker nodes in the machine-api, you will need to [create a new machineSet in OpenShift](openstack-machine-sets).
+
+```sh
+oc get machineset -n openshift-machine-api <machineset_name> -o yaml > sriov_machineset.yaml
+```
+
+When editing an existing machineSet, or a copy of one, to create SR-IOV worker nodes, add each subnet configured for SR-IOV to the machineSet's `providerSpec`. The example below will attach ports from the `radio` and `uplink` subnets, from the previous example, to all the worker nodes in the machineSet. For all SR-IOV ports, always set:
+
+ - `nicType: direct`
+ - `portSecurity:false`
+
+Set `trunk: false` in the machineconfig since trunking is not supported with SR-IOV at this time. Any ports that will be used with SR-IOV must have `portSecurity:false` and `vnicType: direct`. Note that security groups or allowedAddressPairs can not be set on a port when `portSecurity` is disabled. The port's name will be `<machine-name>-<nameSuffix>`, and the `nameSuffix` is required field in the port definition. Tags can optionally be added to ports by adding them to the `tags` list. The example below shows how a machineset can be created to create SR-IOV capable ports on the `Radio` and `Uplink` networks and subnets we defined in a previous example.
```

Comment:
```suggestion
Because trunking is not supported with SR-IOV, set `trunk: false` in the machine config. Any ports that are used with SR-IOV must have `portSecurity:false` and `vnicType: direct`. 

Note that security groups or allowedAddressPairs can not be set on a port if `portSecurity` is disabled. The name of the port will be `<machine-name>-<nameSuffix>`, and the `nameSuffix` is required field in the port definition. 

Optionally, you can add tags to ports by adding them to the `tags` list. The following example shows how a machineset can be created that creates SR-IOV capable ports on the `Radio` and `Uplink` networks and subnets that were defined in a previous example:
```